### DISCUSSION TITLE
Bulleted list Color intensity Meaning in the Dashboard

### DISCUSSION THREAD

**Discussion by cham2019** [mastodon/mastodon#22471] [General]
Hi,
Could someone let me know why the bullet list dots are different in color intensity in the the Dashboard details under Administration?

Specifically the bottom part where it shows the Software and Space Usage basic info.  On my server, Software bullets are all vivid purple.  Space Usage shows the first 2 as "dimmed" bullets (like they were "OFF"), then the last one -Media Storage- is shown vivid purple again.
Why PostgreSQL and REDIS have dimmed bullets?  I would like to learn what does that mean (in case it means something is wrong with my Space Usage).

Thanks
![reason](https://user-images.githubusercontent.com/48077863/208329165-c0b9658b-b35f-4921-a083-0ef67a340478.jpg)

**Comment by mjankowski** [comment]
I realize this is three years old, but -- you can basically ignore those colors.

They share code with other UX elements where the color is sort of useful/meaningful, but in this case its just sort of misleading. You can review the version numbers and space usage metrics, but the bullet colors are not really interesting in this part of the page.
